source/aggregation-pipeline-builder.txt

```diff
@@ -69,9 +69,10 @@ The :pipeline:`$out` stage is not available if you are connected to a
 Save a Pipeline
 ---------------
 
-You can save your pipeline so it can be reloaded into
-the pipeline builder in the future. You can also create a
-:manual:`view </core/views/>` from the results of your pipeline.
+You can save your pipeline to reload and modify the pipeline while
+retaining the original version. You can also
```

Comment:
I feel like this first sentence is slightly less clear than it was previously.

- You can save a pipeline so that you can access it later
- If you load a saved pipeline, you can make changes to it without changing the saved copy

Reply:
Good call. Tried to make this more parallel with the description for loading views but I think I did a bit of over-engineering here.